tools/cli/factory.go

```diff
@@ -225,11 +237,25 @@ func (b *clientFactory) createTLSConfig(c *cli.Context) (*tls.Config, error) {
 	return nil, nil
 }
 
-func fetchCACert(path string) (*x509.CertPool, error) {
-	caPool := x509.NewCertPool()
-	caBytes, err := ioutil.ReadFile(path)
-	if err != nil {
-		return nil, err
+func fetchCACert(path string) (caPool *x509.CertPool, err error) {
+	caPool = x509.NewCertPool()
+	var caBytes []byte
+
+	if strings.HasPrefix(path, "http") {
```

Comment:
I think we should rename `path` to `pathOrUrl` and be a little bit more strict with the prefix by checking both `http://` and `https://` so `http.pem` file doesn't get badly interpreted.

Reply:
👍